pyproject.toml

```diff
@@ -24,7 +24,10 @@ readerwriterlock = "^1.0.6"
 pycryptodomex = "^3.9.6"
 pendulum = "^2.0.5"
 dataclasses = { version = "^0.7", python = ">=3.6,<3.7" }
-gstly = "^0.1.5"
+gstly = [
+    { version = "^0.1.6", markers = "sys_platform != 'win32'" },
+    { version = "^0.1.6", markers = "sys_platform == 'win32'", optional = true },
```

Comment:
and just to confirm, it wasn't possible to put a git dependency here for sys_platform==win32, or you decided against it?

Reply:
As this is the open-source repo, I deemed that to not be a good solution. No one, even linux users, would be able to work with the poetry stuff

Reply:
Wait, why wouldn't it have worked for linux users? That's odd

Reply:
They wouldn't be able to do things like update the lock file without having an SSH key

Reply:
Are you sure that's not the case with having it in the `--extras`? I don't see why poetry would suddenly not need to investigate those dependencies

Reply:
The lock contains _everything_, even things in extras. If you notice, I didn't put the git dependency in extras. I'm just relying on our build system doing in manually